docs/QSG/tour.md

```diff
@@ -308,7 +313,7 @@ bring-up DAOS servers and clients.
 
 	use 'daos help RESOURCE' for resource specifics
 
-	$ daos help cont
+	$ daos cont --help
 	daos command (v1.2), libdaos 1.2.0
```

Comment:
There is no version info so maybe remove this line and update the help?

` daos cont --help
Usage:
  daos [OPTIONS] container <command>

Application Options:
      --debug     enable debug output
      --verbose   enable verbose output (when applicable)
  -j, --json      enable JSON output

Help Options:
  -h, --help      Show this help message

Available commands:
  check          check objects' consistency in a container
  clone          clone a container
  create         create a container
  create-snap    create container snapshot (aliases: snap)
  del-attr       delete container user-defined attribute (aliases: delattr)
  delete-acl     delete a container's ACL
  destroy        destroy a container
  destroy-snap   destroy container snapshot
  get-acl        get a container's ACL
  get-attr       get container user-defined attribute (aliases: getattr)
  get-prop       get container user-defined attribute (aliases: getprop)
  list           list all containers in pool (aliases: ls)
  list-attr      list container user-defined attributes (aliases: list-attrs, lsattr)
  list-objects   list all objects in container (aliases: list-obj)
  list-snap      list container snapshots (aliases: list-snaps)
  overwrite-acl  replace a container's ACL (aliases: replace)
  query          query a container
  set-attr       set container user-defined attribute (aliases: setattr)
  set-owner      change ownership for a container (aliases: chown)
  set-prop       set container user-defined attribute (aliases: setprop)
  stat           get container statistics
  update-acl     update a container's ACL
`